cfep-23.md

```diff
@@ -0,0 +1,73 @@
+- The thread system offers very limited ways of tracking conversations and making them discoverable. It's very difficult to follow on topics started hours ago, let alone days or weeks ago.
+- Scoped discussions require a lot of discipline from the users, since threads are optional, and the real-time chat is preferred in the UI.
+
+Zulip, on the other side, is a free to use, open source, real-time discussion board. It has web, desktop and mobile clients. Each community has its Zulip instance, where several "channels" can be created. Each channel hosts a number of threads or topics. Instant messaging is also available within the Zulip instance. Users can also create a new account or log in with their Github credentials, among others.
+
+Zulip offers some features we would consider beneficial to the conda-forge community:
+
+- Threads (or topics) are encouraged and the default user experience.
+- Each topic must have a title that promotes well-scoped discussions, which can be marked as solved when the issue has been addressed.
+- Moderation tools are available to split threads into new ones in case the conversation goes off-topic.
+- Users can "follow" threads they find interesting, which has its own dedicated view. This makes it easier to track updates to conversations you have started or participated in.
+- Private rooms are also available, but they don't offer end-to-end encryption. For this matter, a sensitive communications protocol is proposed below.
+
+## Proposal
+
+After accepting this proposal, we will implement the following decisions:
+
+- The Zulip instance published at `conda-forge.zulipchat.com` will be open to the public with world-readable permissions.
+- The General channel will be the replacement for the `conda-forge/conda-forge.github.io` chat room. This will be reflected in the conda-forge.org documentation, and reminded periodically in the Element room.
+- The conda-forge core team will move its private communications to the Zulip instance, and stop using the Element rooms. A last message will be published in the Element core chat room with the details on how to transition to the Zulip instance. 
+- New channels will be created in the instance as considered necessary (e.g. infrastructure, compilers, etc).
+- Sensitive topics will NOT be discussed directly in the Zulip instance. Instead they will follow the protocol described below.
+- Some Element rooms related to conda-forge (e.g., rooms where we interact with other communities like bioconda) will remain unchanged as needed. We will however strive to move as much of the conda-forge Element space to Zulip as possible.
+
+Three months after the approval of this CFEP, the core team will consider the migration finished and the public rooms will become private and archived. If the core team estimates that there's still sufficient activity, another three months will be granted, with periodic reminders of the new Zulip instance. After that period, no more extensions will take place.
```

Comment:
Think we would want the history of the public room to remain readable. Would making it private hinder that?

Reply:
As per Matrix terminology:

<img width="739" alt="image" src="https://github.com/user-attachments/assets/14538f40-dbe4-4c6d-aa34-6e4ce0dcd651">


<img width="694" alt="image" src="https://github.com/user-attachments/assets/a5717045-d837-43f9-a194-69acabf0f68a">


So I'm assuming that private room is the closest we get to read-only (if "Who can read history" is set to "Anyone").